CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ endif()
 if(ARM_DYNAREC)
     add_definitions(-DDYNAREC)
     add_definitions(-DARM)
+    add_definitions(-march=armv7-a+fp -mfpu=neon -mfloat-abi=hard -marm)
```

Comment:
I cannot accept that. The Pandora is not hardhf abi for example. Other platform may want some other arch also, or fpu. Only `-marm` is strickly universal and needed.
But you can create a new option for "GameShell" for example, with this set of best flags.